### ISSUE TITLE
Build fails on 2020-10-04

### ISSUE THREAD

**Issue by zesterer** [yoanlcq/vek#60]
```
   Compiling vek v0.12.0 (/home/joshua/Projects/vek)
error: expected `,`
   --> src/bezier.rs:428:30
    |
428 |             pub start: $Point<T>,
    |                              ^
...
834 |     impl_all_beziers!{repr_simd}
    |     ---------------------------- in this macro invocation
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected `,`
   --> src/bezier.rs:428:30
    |
428 |             pub start: $Point<T>,
    |                              ^
...
834 |     impl_all_beziers!{repr_simd}
    |     ---------------------------- in this macro invocation
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected `,`
   --> src/bezier.rs:612:30
    |
612 |             pub start: $Point<T>,
    |                              ^
...
834 |     impl_all_beziers!{repr_simd}
    |     ---------------------------- in this macro invocation
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected `,`
   --> src/bezier.rs:612:30
    |
612 |             pub start: $Point<T>,
    |                              ^
...
834 |     impl_all_beziers!{repr_simd}
    |     ---------------------------- in this macro invocation
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected `,`
   --> src/bezier.rs:428:30
    |
428 |             pub start: $Point<T>,
    |                              ^
...
839 |     impl_all_beziers!{repr_c}
    |     ------------------------- in this macro invocation
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected `,`
   --> src/bezier.rs:428:30
    |
428 |             pub start: $Point<T>,
    |                              ^
...
839 |     impl_all_beziers!{repr_c}
    |     ------------------------- in this macro invocation
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected `,`
   --> src/bezier.rs:612:30
    |
612 |             pub start: $Point<T>,
    |                              ^
...
839 |     impl_all_beziers!{repr_c}
    |     ------------------------- in this macro invocation
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected `,`
   --> src/bezier.rs:612:30
    |
612 |             pub start: $Point<T>,
    |                              ^
...
839 |     impl_all_beziers!{repr_c}
    |     ------------------------- in this macro invocation
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 8 previous errors

error: could not compile `vek`

To learn more, run the command again with --verbose.
```

**Comment by yoanlcq** [comment]
Hi, thanks for reporting this, I'll look into this ASAP.

**Comment by zesterer** [comment]
Thanks. This is confusing the heck out of me. `cargo expand` doesn't seem to show any bizarre code generation going on.

**Comment by yoanlcq** [comment]
Whoops I didn't mean to close this yet. [I've submitted a workaround here](https://github.com/yoanlcq/vek/commit/f756410fbba4b344ed64ab9a456bb93d0e79f485). I'd like to see if CI builds successfully as well.

The workaround was to put the structs in their own isolated module, and then define a `type Point<T> = $Point<T>`, then replace all refs to `$Point<T>` with `Point<T>`.

However I do think that we should report this as a bug to the relevant team; basically, an update broke the build "for free" and I suspect too few people had the same problem.

**Comment by yoanlcq** [comment]
Well this looks fixed. One AppVeyor job choked on a connection error for whatever reason.
I've published version `0.12.1` with the fix, hopefully this does the trick.

About reporting the bug to the Rust repo, I wish to do it sometime soon, provided that I don't forget about it...

**Comment by zesterer** [comment]
Thanks very much for the responsive fix!
